relationships/synthesis/EXT-SERVICE-to-KAFKATOPIC.yml

```diff
@@ -20,7 +22,71 @@ relationships:
           fields:
             - field: clusterName
               capture:
-                attribute: server.address            
+                attribute: net.peer.name            
+                regex: "^[^\\.]+\\.([^\\.]+)\\.[^\\.]+\\.[^\\.]+\\.kafka\\.[^\\.]+\\.amazonaws\\.com:[0-9]+.*"
```

Comment:
Similar to my question above, should this rule be constrained by adding a condition that makes it specific to AWS?

Reply:
Probably being too pragmatic, but if the domain includes `amazonaws...com` there is no way it will not be specific to AWS. Alternatively, you can add and check a specific tag for this purpose.